recipes/maplab/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "maplab" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/maplab-{{ version }}.tar.gz
+  sha256: ff4257382a75fe601a78c24eb901ace343a53001aff0e967b14eac116a237be6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - folium
+    - ipyleaflet
+    - pandas
+    - openpyxl
+    - httpx
+    - geopandas
+
+test:
+  imports:
+    - maplab
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/hwilli72/maplab
+  summary: Python Boilerplate contains all the boilerplate you need to create a Python package.
```

Comment:
Can you add something that describes this package here?

Reply:
Yes, just added the description.